docs/edge.rst

```diff
@@ -0,0 +1,253 @@
+.. _edge:
+
+============
+CrateDB Edge
+============
+
+Crate.io is pleased to `announce CrateDB Edge`_, the hybrid cloud database
+solution integrating local control over regions and data storage with the
```

Comment:
not sure about phrasing it "local control over regions" I think it is rather "remote control over local regions".

Reply:
I would change the focus of the control from the region to the CrateDB clusters to avoid saying things the customer may understand differently than intended. What about:

"integrating control of CrateDB clusters running in on premises or customer controlled cloud infrastructure"